keras_retinanet/bin/train.py

```diff
@@ -305,6 +307,13 @@ def main(args=None):
     # create the generators
     train_generator, validation_generator = create_generators(args)
 
+    if args.backbone == 'resnet':
+        from ..models.resnet import resnet_retinanet as retinanet, custom_objects, download_imagenet
+    elif args.backbone == 'mobilenet':
+        from ..models.mobilenet import mobilenet_retinanet as retinanet, custom_objects, download_imagenet
+    else:
+        raise NotImplementedError('Backbone model not implemented.')
```

Comment:
Also print the `backbone` here, maybe something like:
```python
raise NotImplementedError('Backbone \'{}\' not implemented.'.format(args.backbone))
```

Reply:
done!